packages/core/src/html/NodeProcessor.js

```diff
@@ -315,6 +321,60 @@ class NodeProcessor {
     return node;
   }
 
+  /*
+   * Syntax is (space)#char#(any text)##(space)
+   * If we have #char# without ## or ## without #char# we don't replace
+   * To escape start do \#char#, so it will show up as #char# (escaping also works with any front char)
+   * To escape end do \##, which will show up as ##
+   */
+  preprocessMarkdown(content) {
+    const lines = content.split('\n');
+    let amendedContent = '';
```

Comment:
"Amend" feels like it's saying the original content is "incorrect", perhaps we can use a more neutral name here, like "result", or "preprocessedContent"? Ok to leave it as is too.